package.json

```diff
@@ -8,7 +8,7 @@
     "deduplicate": "pnpm dedupe",
     "benchmark:browser": "pnpm --filter benchmark browser",
     "build": "lerna run build --ignore docs",
-    "build:ci": "lerna run build --ignore docs --concurrency 8 --skip-nx-cache",
+    "build:ci": "lerna run build --ignore docs --ignore @app/next-app --concurrency 8 --skip-nx-cache",
```

Comment:
Not strictly necessary, but if we could rename `@app/next-app` to `@app/pigment-css-next-app`, that would help clarify the ignore.

Reply:
Didn't want to change too much the commands that are already in @brijeshb42 muscle memory